site/en/blog/more-mv2-transition/index.md

```diff
@@ -25,18 +25,26 @@ Chrome will take a gradual and experimental approach to turning off Manifest V2
 
 Specifically:
 
-* Starting in January in **Chrome 112**, Chrome may run experiments to turn off support for Manifest V2 extensions in Canary, Dev, and Beta channels.
-* Starting in June in **Chrome 115**, Chrome may run experiments to turn off support for Manifest V2 extensions in all channels, including stable channel.
+* ~~Starting in January in **Chrome 112**, Chrome may run experiments to turn off support for Manifest V2 extensions in Canary, Dev, and Beta channels.~~
```

Comment:
Why leave them in?

Reply:
I think it makes sense from an archival point of view. It would also break the flow of the blog post if we removed these, which would mean we'd need to make additional changes. I really wanted to keep this as a simple PR since it doesn't feel worth spending too much time on.